draft-ietf-tls-tls13.md

```diff
@@ -1516,55 +1530,68 @@ derivation scheme for all cipher modes.
 The basic TLS Handshake for DH is shown in {{tls-full}}:
 
 ~~~
-     Client                                               Server
-
-     ClientHello
-       + KeyShare              -------->
-                                                     ServerHello
-                                                      + KeyShare
-                                           {EncryptedExtensions}
-                                          {ServerConfiguration*}
-                                                  {Certificate*}
-                                           {CertificateRequest*}
-                                            {CertificateVerify*}
-                               <--------              {Finished}
-     {Certificate*}
-     {CertificateVerify*}
-     {Finished}                -------->
-     [Application Data]        <------->      [Application Data]
-
-            +  Indicates extensions sent in the
-               previously noted message.
-
-            *  Indicates optional or situation-dependent
-               messages that are not always sent.
-
-            {} Indicates messages protected using keys
-               derived from the ephemeral secret.
-
-            [] Indicates messages protected using keys
-               derived from the master secret.
+       Client                                               Server
+  
+Key  / ClientHello                                                  
+Exch \  + KeyShare               -------->                           
+                                                       ServerHello  \ Key
+                                                        + KeyShare  / Exch
+                                             {EncryptedExtensions}  ^ 
+                                            {ServerConfiguration*}  | Server
+                                             {CertificateRequest*}  v Params
```

Comment:
The slides you showed in Yokohama had the CertificateRequest before the ServerConfiguration.  I think that it belongs there more properly (since it includes information that the client needs to remember for 0RTT).